docs/components/ToggleSwitchDocs.js

```diff
@@ -5,6 +5,14 @@ const { ToggleSwitch } = require('mx-react-components');
 const Markdown = require('components/Markdown');
 
 class ToggleSwitchDocs extends React.Component {
+  state = {
+    checked: false
+  }
```

Comment:
we probably don't need to set this explicitly to false - but if you like to for discoverability that's 🆒 too